static/_redirects

```diff
@@ -132,8 +132,11 @@
 /docs/contribute/review-issues/     /docs/home/contribute/review-issues/ 301
 /docs/contribute/stage-documentation-changes/     /docs/home/contribute/stage-documentation-changes/ 301
 /docs/contribute/style-guide/     /docs/home/contribute/style-guide/ 301
-
 /docs/contribute/write-new-topic/        /docs/home/contribute/write-new-topic/ 301
+/docs/contribute/start/        /docs/contribute/ 301
+/docs/contribute/intermediate/        /docs/contribute/ 301
```

Comment:
@kbhawkey I added two redirects here, but the implication of this reorganization is that there's no directly equivalent section to redirect to. I'm not sure these are useful?

Reply:
OK.